apps/docs/content/guides/auth/auth-hooks.mdx

```diff
@@ -712,24 +1168,1537 @@ revoke execute
 </TabPanel>
 </Tabs>
 
-## Local Development
+**HTTP Examples**
 
-Unlike the hosted platform, the local development dashboard does not have a user interface for linking a Hook. Instead, edit `config.toml` to link an Auth Hook and run it with your local setup. Modify the `auth.hook.<hook_name>` field and set `uri` to a value of `pg-functions://postgres/<schema>/<function_name>`
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="http-add-metadata-claim"
+  queryGroup="language"
+>
+<TabPanel id="http-add-metadata-claim" label="Add claim">
 
-For instance, the `config.toml` for a Custom Access Token Hook which uses the function `public.custom_access_token_hook`:
+Your company wishes to add assign permissions via the role claim on the `app_metadata` field. Add the role claim to the token via a Hook.
 
+```javascript
+import { Webhook } from 'https://esm.sh/standardwebhooks@1.0.0'
+import { readAll } from 'https://deno.land/std/io/read_all.ts'
+import * as base64 from 'https://denopkg.com/chiefbiiko/base64/mod.ts'
+
+Deno.serve(async (req) => {
+  const payload = await req.text()
+  const base64_secret = Deno.env.get('AUTH_CUSTOM_ACCESS_TOKEN_SECRET')
```

Comment:
Maybe drop the AUTH_ prefix since we don't specify it the example where we introduce the secret